nservicebus/upgrades/absdatabus-4to5.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Azure Blob Storage Databus Upgrade Version 4 to 5
+reviewed: 2020-10-29
+component: ABSDataBus
+related:
+ - nservicebus/upgrades/7to8
+isUpgradeGuide: true
+upgradeGuideCoreVersions:
+ - 8
+---
+
+Nothing to report...
```

Comment:
I know this needs some love here, but I wasn't sure what to add here. There will be meaningful changes from 3 to 4, but we don't have any changes as of now, between 4 to 5, except the impact created by Core v8. Thoughts?

Reply:
Then we [keep it real](https://www.youtube.com/watch?v=20ob9meWOC4)
```suggestion
Upgrade from version 7 to 8 does not require any code changes.
```